setup.py

```diff
@@ -295,25 +295,34 @@ def has_ext_modules(self):
         ]
     },
     install_requires=[
-        'numpy>=1.16.1',
+        'numpy>=1.19',
+        'pybind11>=2.7',
         'scipy>=1.2.1',
         'msgpack>=0.5.6',
         'msgpack-numpy>=0.4.4',
         'pyzmq>=22.3.0',
         'pyFAI>=0.17.0',
-        'PyQt5==5.13.2',
+        'PyQt5>=5.13.2',
         'PyQt5-sip>=12.9.0',
         'EXtra-data>=1.0.0',
         'EXtra-geom>=0.8.0',
         'karabo-bridge>=0.5.0',
-        'toolz>=0.9.0',
-        'silx>=0.9.0',
-        'hiredis==1.0.1',
-        'redis==3.5.2',
+        'hiredis>=1.0.1',
+        'redis>=3.5.2',
         'psutil>=5.6.2',
-        'imageio==2.8.0',
-        'Pillow==8.3.2',
+        'imageio>=2.8.0',
+        'Pillow>=8.3.2',
         'pyyaml>=5.2',
+        # These dependencies are not directly used, but are needed to satisfy
+        # pip's resolver:
+        'pygments',
+        'jinja2',
+        'decorator',
+        'pexpect>4.3',
+        'pygments',
+        'toolz>=0.8.2',
+        'packaging',
+        'cffi>=1'
```

Comment:
You don't actually need to do this for dependabot to work, since the configuration is:

https://github.com/European-XFEL/EXtra-foam/blob/a93a9540ca609e164a532a26afa8fd2391a6a38b/.github/dependabot.yml#L1-L6

Where the `.github/dependabot/` directory only has the `constraints.txt` file in it.

And the install step for the tests is:

https://github.com/European-XFEL/EXtra-foam/blob/a93a9540ca609e164a532a26afa8fd2391a6a38b/.github/workflows/tests.yml#L62

That means that dependabot will bump the constraints files, create a PR, and the tests execute with the bumped dependency versions.

Reply:
Ah hah, good point. Fixed in da95b2e7.